tier2/{{cookiecutter.project_slug}}/repometrics/hooks/post_gen_project.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import time 
```

Comment:
Why is time imported?

Reply:
was doing some local test runs and forgot to remove it, should be deleted now!